code/style.css

```diff
@@ -1,7 +1,166 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  background-color: black;
+  font-family: 'IBM Plex Sans KR', sans-serif;
+  font-weight: bold;
+  margin: 0;
+  text-align: center;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+header {
+  background-image: url(https://i.ibb.co/wY0sPt0/egor-myznik-ZZ1-Zr-HKWASY-unsplash.jpg);
+  background-size: cover;
+  height: 500px;
+  background-position: center;
+  margin-bottom: 50px;
+}
+
+.container {
+  margin-top: 25%;
+}
+
+a {
+  font-family: 'Poppins', sans-serif;
+  text-decoration: none;
+  color: rgb(135, 133, 133);
+  font-weight: bold;
+}
+
+a:hover {
+  color: black;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  color: whitesmoke;
+  font-size: 1cm;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  color: whitesmoke;
+  font-size: 0.7cm;
+}
+
+p {
+  color: whitesmoke;
+}
+
+.accordion {
+  margin-bottom: 40px;
+  margin-top: 25px;
+}
+
+div {
+  color: #444944;
+  padding-bottom: 15px;
+  display: block;
+}
+
+div:hover {
+  cursor: pointer;
+  color: whitesmoke;
+}
+
+.answer {
+  display: none;
+  color: rgb(135, 133, 133);
+  padding: 0px 40px 15px 40px;
+  text-align: left;
+  margin-left: 40px;
+}
+
+.question + .answer {
+  display: block;
```

Comment:
I think this overwrites the styling above and keeps the .answer class visible. If this is what you intend I think you can write display: block; in the class above instead of display: none; and just remove these lines.

However if you intend to have the .answer hidden from the start you need to use some other selector here. Maybe a .active + .answer selector and then toggle .active in the javascript?